CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Changelog for Elixir v1.8
 
+Elixir v1.8 comes with many improvements at the infrastructure level, improving compilation time, speeding up common patterns and adding features around introspection of the system.
+
+## Custom struct inspections
+
+Elixir now provides a derivable implementation of the `Inspect` protocol. In a nutshell, this means it is really easy to filter data from your data structures whenever they are inspected. For example, imagine you have a user struct with security and privacy sensitive information:
+
+```elixir
+defmodule User do
+  defstruct [:id, :name, :age, :email, :encrypted_password]
+end
+```
+
+By default, if you inspect a user via `inspect(user)`, it will include all fields. This can cause fields such as `:email` and `:encrypted_password` to appear in logs, error reports, etc. You could always define a custom implementation of the `Inspect` protocol for such cases but Elixir v1.8 makes it simpler by allowing you to derive the `Inspect` protocol:
+
+```elixir
+defmodule User do
+  @derive {Inspect, only: [:id, :name, :age]}
+  defstruct [:id, :name, :age, :email, :encrypted_password]
+end
+```
+
+You can also pass `@derive {Inspect, except: ...}` in case you want to keep all fields by default and exclude only some.
+
+## TimeZone database support
```

Comment:
```suggestion
## Time zone database support
```

Reply:
@eksperimental It's the button at the left of the comment's toolbar:

<img width="819" alt="screen shot 2018-12-21 at 18 31 43" src="https://user-images.githubusercontent.com/651203/50355374-c27bec00-054e-11e9-86b2-0898e5778905.png">

Pretty handy!

Reply:
Aweeesome. I just used it. Thank you very much